### PR TITLE
MPDX-8154 Add Contact Button and fix Star filtering

### PR DESCRIPTION
**Pull request by dr-bizz**
## Description
In this PR, I fix the star filtering on the Flows view. I also added the "Add Contact to Appeal" button to the asked column. Refreshing the contacts queries once contacts are added.

## Checklist:

- [ ] I have given my PR a title with the format "MPDX-(JIRA#) (summary sentence max 80 chars)"
- [ ] I have applied the appropriate labels. (_Add the label "On Staging" to get the branch automatically merged into staging._)
- [ ] I have requested a review from another person on the project

### CODE REVIEW

File: src/components/Tool/Appeal/Flow/ContactFlowColumn/ContactFlowColumn.test.tsx [hunk 1]
```diff
@@ -95,4 +101,24 @@ describe('ContactFlowColumn', () => {
       getByRole('menuitem', { name: 'Select 1 contact' });
     });
   });
+
+  it('should have "Add Contact to Appeal" button', async () => {
```
Comment (canac) [inline src/components/Tool/Appeal/Flow/ContactFlowColumn/ContactFlowColumn.test.tsx:105]:
```suggestion
  it('asked column should have "Add Contact to Appeal" button', async () => {
```

File: src/components/Tool/Appeal/Flow/ContactFlowColumn/ContactFlowColumn.test.tsx [hunk 2]
```diff
@@ -95,4 +101,24 @@ describe('ContactFlowColumn', () => {
...
+    );
+
+    expect(await findByText('Add Contact to Appeal')).toBeInTheDocument();
+
+    userEvent.click(await findByText('Add Contact to Appeal'));
+
+    expect(await findByText('Add Contact(s) to Appeal')).toBeInTheDocument();
+  });
+
+  it('should not have the "Add Contact to Appeal" button', async () => {
```
Comment (canac) [inline src/components/Tool/Appeal/Flow/ContactFlowColumn/ContactFlowColumn.test.tsx:117]:
```suggestion
  it('other columns should not have the "Add Contact to Appeal" button', async () => {
```

**Review by canac — APPROVED** [review]
Love it! 🔥

File: src/components/Tool/Appeal/Flow/ContactFlowColumn/ContactFlowColumn.test.tsx (hunk 1 above)
Comment (dr-bizz) [reply src/components/Tool/Appeal/Flow/ContactFlowColumn/ContactFlowColumn.test.tsx:105]: I've change it to the following so it flows with the `it` at the beginning.

```it('should ensure the "asked" column has the "Add Contact to Appeal" button', async () => {```

Comment (dr-bizz) [reply src/components/Tool/Appeal/Flow/ContactFlowColumn/ContactFlowColumn.test.tsx:105]: Actually, I don't like how it reads, as it makes it more confusing. I'll do what you suggested.

**Pull request merged**